target/snitch_cluster/sw/apps/snax-block-gemm/data/params.hjson

```diff
@@ -0,0 +1,30 @@
+// Copyright 2023 KU Leuven.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+//
+// Xiaoling Yi <xiaoling.yi@esat.kuleuven.be>
+
+// Parameters for a block GEMM
+// need to fix batch = 1
+
+// large martix multiply 4 * 8 * 4 with no input data contention
+{
+    Batch: 1
+    M: 2,
+    N: 2,
+    K: 2,
+    strideInnermostA: 256,
+    strideInnermostB: 256,
+    strideInnermostC: 256,
```

Comment:
This variable is not mentioned in your docs here:
https://github.com/KULeuven-MICAS/snax-gemm#computation-analysis-1
Neither are `delta_local_a` and `delta_local_b`?

Reply:
This stride is added after the doc. The new doc will include this!